Tools/autotest/rover.py

```diff
@@ -303,6 +303,41 @@ def test_sprayer(self):
         if ex:
             raise ex
 
+    def drive_max_rcin(self, timeout=30):
+        """Test max RC inputs"""
+        self.context_push()
+        ex = None
+
+        try:
+            self.progress("Testing max RC inputs")
+            self.wait_mode("MANUAL")
+
+            self.wait_ready_to_arm()
+            self.arm_vehicle()
+
+            self.set_rc(3, 2000)
+            self.set_rc(1, 1000)
+
+            tstart = self.get_sim_time()
+            while self.get_sim_time() < tstart + timeout:
```

Comment:
Have to be careful with `get_sim_time` as it does swallow messages.  This is why we have `get_sim_time_cached`.

Reply:
Changed to `get_sim_time_cached()`